### PR TITLE
Licence it

### PR DESCRIPTION
**Pull request by ticosax**
Licensing it and change README syntax to rst

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
+pyipptool
+=========
+
+.. image::
+   https://magnum.travis-ci.com/ezeep/pyipptool.png?token=1cZ3Jy2DStcoTAvezSS8&branch=master
+   :target: https://magnum.travis-ci.com/ezeep/pyipptool
+
+Convenient ipp request generator to interrogate CUPS or IPP devices with help
```
Comment (sbellem) [inline README.rst]: would write: "...with the help ..."

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+.. code-block:: console
+
+   python setup.py test
+
+Configuration
+-------------
+
+Either in ``~/.pyipptool.cfg`` or in ``/etc/pyipptool/pyipptol.cfg``
+add the following content
```
Comment (sbellem) [inline README.rst]: re-wording suggestion:

Add the following content in `~/.pyipptool.cfg` or `/etc/pyipptool/pyipptol.cfg`

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+.. code-block:: configuration
+
+    [main]
+    ipptool_path = /usr/bin/ipptool
+    ;If authentication is required
+    login = admin
+    password = secret
+
+With the path to your installed ipptool
```
Comment (sbellem) [inline README.rst]: where `ipptool_path` points to the absolute path of your installed ipptool

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Usage
+-----
+
+Create infinite time subscription for printer-XYZ class for ``rss`` notifier
```
Comment (sbellem) [inline README.rst]: Create an infinite time subscription for printer-XYZ class for an `rss` notifier

Comment (ticosax) [inline README.rst]: thx

File: README.rst (hunk 3 above)
Comment (ticosax) [inline README.rst]: thx

File: README.rst (hunk 2 above)
Comment (ticosax) [inline README.rst]: thx

File: README.rst (hunk 1 above)
Comment (ticosax) [inline README.rst]: thx

**Comment by marianzange** [comment]
:yellow_heart: Ok!

**Pull request merged**